styles.css

```diff
@@ -62,61 +82,61 @@ nav {
 
 .cv-download-button:hover {
 	background-color: #048230;
-	color: #cfd2d4;
+	color: var(--my-white);
 }
 
 header {
 	position: relative;
-	text-align: center;
 }
 
-.header-title {
-	position: absolute;
-	top: 50%;
-	left:50%;
-	transform: translate(-50%, -50%);
-	border: 2px solid;
-	order: 1;
-	padding: 5px 100px;
-	font-size: 4rem;
-	text-transform: uppercase;
-	white-space: nowrap;
-}
+
 
 main {
 	margin-top: 20px;
 	display: grid;
 	grid-template-columns: 2fr 1fr;
 	max-width: 2000px;
-	height: auto;
-	position: relative;
+	padding: 20px;
 }
 
 header img {
 	width: 100%;
 }
 
 .portfolio {
+	display: flex;
```

Comment:
it looks like the normal document flow now does what you need for the portfolio section. This means you don't need to put flex on it.